app/assets/stylesheets/components/team_members.scss

```diff
@@ -2,7 +2,7 @@
   margin-bottom: -5px;
 }
 
-.govuk-link {
+.govuk-single-link {
   display: block;
 }
```

Comment:
*Please read comments on `_table.html.erb` first and then come back here.*

Revert this back to `govuk-link` and scope `.list-item-padding` and `.govuk-link` to be within `membership-list` class.

E.g.
```
.membership-list {
  .list-item-padding { ... }
  .govuk-link { ... }
}
```